UNIQUE_SETUPS.md

```diff
@@ -18,9 +18,7 @@ To use `xrpl.js` with React, you need to install shims for core NodeJS modules.
 1. Install shims (you can use `yarn` as well):
 
    ```shell
-   npm install --save-dev \
-       buffer \
-       process
+   npm install --save-dev process
```

Comment:
The integration tests need process to be polyfilled but I dont think running it regularly does.  I need to come up with a good way to make it clear that you shouldnt need any polyfills with typical operation.

Reply:
If it is only necessary for testing, I wouldn't include it in these steps. Instead I'd just make a note in the `CONTRIBUTING.md`. 

(Is there a reason we don't include `process` as a dev dependency to automatically install it for people testing, or would that conflict with another package we use?

Reply:
Resolved by 01dce8581da25469b7a436d0c2af5e14559ff8b3